src/ProjectTemplates/Microsoft.McpServer.ProjectTemplates/templates/McpServer-CSharp/remote/McpServer-CSharp.csproj-in

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net10.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
```

Comment:
I forgot to remove the `TargetFrameworkOverride` content from the remote flavor.

```suggestion
    <TargetFramework>net10.0</TargetFramework>
```